specification/consumption/resource-manager/Microsoft.Consumption/stable/2018-03-31/examples/Tags.json

```diff
@@ -0,0 +1,47 @@
+{
+  "parameters": {
+    "api-version": "2018-03-31",
+    "billingAccountId": "12345"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "providers/Microsoft.CostManagement/billingAccounts/{billingaccount-id}/providers/Microsoft.Consumption/tags/tags1",
+        "name": "tags1",
+        "type": "Microsoft.Consumption/tags",
+        "eTag": "\"1d34d012214157f\"",
+        "properties": {
+          "tags": [
```

Comment:
The property name here should be "costTags", as defined in `TagProperties`.

Reply:
Thank you! for mentioning this. Actually it was wrong at the other place. I am corrected it there.